conjure-test/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "conjure-test"
-version = "2.1.0"
+version = "2.2.0"
```

Comment:
Should this be 3.2.0?

Reply:
That crate isn't published so the version doesn't matter. It just ends up getting bumped when I use [cargo-edit](https://crates.io/crates/cargo-edit) to bump the crate versions. I should probably sync them up just to avoid confusion.